### PR TITLE
Aaa dev

### PR DESCRIPTION
**Pull request by dmielaikaite**
paths were updated and fixed parsing issue of EU redirectors

### CODE REVIEW

File: AAAOps/Federation/create_allow-list.sh [hunk 1]
```diff
@@ -82,14 +82,11 @@ echo "* redirect cms-xrd-transit.cern.ch+:1213" >> $FEDINFO/out/list_eu.allow
 
 #Quick fix for "[" character in prod.txt 
 cat $FEDINFO/in/prod.txt | awk '{ if ($1 == "[") print "Unknown.Host"; else print $1;}' > $FEDINFO/in/tmp
-cp $FEDINFO/in/tmp $FEDINFO/in/prod.txt
+scp $FEDINFO/in/tmp $FEDINFO/in/prod.txt
```
Comment (zvada) [inline AAAOps/Federation/create_allow-list.sh]: Why `scp`?

File: AAAOps/Federation/create_allow-list.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-BASE=/root
-FEDINFO=/root/FederationInfo
+BASE=/root/
```
Comment (zvada) [inline AAAOps/Federation/create_allow-list.sh:3]: do we need `BASE` at all?

File: AAAOps/Federation/create_allow-list.sh (hunk 1 above)
Comment (dmielaikaite) [reply AAAOps/Federation/create_allow-list.sh]: oh.. yes, scp is not needed in this case.

File: AAAOps/Federation/create_allow-list.sh (hunk 2 above)
Comment (dmielaikaite) [reply AAAOps/Federation/create_allow-list.sh:3]: If we would like to move all temp files somewhere else it's easier to change the path in one place that go through the code and update each line. this is only one reason why I left it as it is.

File: AAAOps/Federation/create_allow-list.sh (hunk 1 above)
Comment (dmielaikaite) [reply AAAOps/Federation/create_allow-list.sh]: i have updated the code and pushed it again.

Comment (zvada) [reply AAAOps/Federation/create_allow-list.sh]: LGTM

File: AAAOps/Federation/create_allow-list.sh (hunk 2 above)
Comment (zvada) [reply AAAOps/Federation/create_allow-list.sh:3]: oh yeah, sounds good, let's keep it for now.

**Pull request merged**